resources/measures/HPXMLtoOpenStudio/resources/dr_cl_setup.csv

```diff
@@ -0,0 +1,8760 @@
+0
```

Comment:
I would use more consistency/description in how you name these csv resource files.